lib/app/frontend/FrontendProcess.js

```diff
@@ -55,12 +56,47 @@ class FrontendProcess {
     })
   }
 
+  /**
+   * Sleep N ms
+   * @param ms
+   * @returns {*}
+   */
+  async sleep (ms) {
+    return new Promise((resolve) => {
+      setTimeout(resolve, ms);
+    })
+  }
+
   /**
    * Runs the Webpack Dev Server.
    */
   async webpackDevServer () {
+    const appFolder = this.appSettings.getApplicationFolder();
+    const themePath = resolve(appFolder, 'themes', this.options.theme)
+
+    /**
+     * Use theme own webpack server implementation if server config file is present
+     */
+    if (existsSync(resolve(themePath, 'webpack.config.js'))) {
```

Comment:
Right now it's not possible to use your 2nd pull request together with the latest `platform-sdk 1.10.0`. That causes a direct dependency between the sdk and Engage. After merging this PR, we'd need to release an @alpha version of the sdk to enable development of pwa 6.11.0

![Bildschirmfoto 2019-11-27 um 08 51 23](https://user-images.githubusercontent.com/13087792/69704105-201b9f80-10f3-11ea-896c-d1652d58c2b6.png)